db/osqlrepository.c

```diff
@@ -36,11 +36,11 @@ struct osql_repository {
 
     hash_t *rqs; /* hash of outstanding requests */
     hash_t *rqsuuid;
-    pthread_rwlock_t hshlck; /* protect the hash */
-
+    pthread_mutex_t hshlck; /* protect the hash */
+    pthread_mutex_t cancelall_mtx; /* cancelall mutex */
```

Comment:
Where is this being used?

Reply:
Merge error, used to be here--Dorin removed, good catch. Thanks